phpunit/FuzzyTestSuiteLoader.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * This file is derived from the PHPUnit project and customized for the
+ * needs of the Gutenberg project.
```

Comment:
This comment is explanatory, but not that useful. That the code here derived from PHPUnit it interesting but it doesn't tell me why it was derived or why that matters. The fact that it serves the needs of the Gutenberg project should be self-evident, since it's part of the Gutenberg project. What are those needs? How are the different from PHPUnit?

The first comment at the top of a module _can_ orient the reader to understand the overall purpose of the module and why it exists.

```php
/**
 * Implements a custom PHPUnit loader so that it's possible to run WordPress
 * tests (with their non-standard filenames) without further modifications.
 * 
 * WordPress test files follow lowercase-kebab-casing conventions:
 *   phpunit/html/wp-html-tag-processor-test.php
 * 
 * PHPUnit test files follow Title_Snake_Casing conventions:
 *   phpunit/html/WP_HTML_Tag_Processor_Test.php
 *
 * This makes it difficult to run the WordPress files from the CLI with
 * PHPUnit. The loader in this module lets PHPUnit find and use both
 * kinds of test file naming schemes.
 *
 * The code mostly derives from the PHPUnit project
 * @see {LINK}
 */
```

Reply:
This is a really good comment, I just added it to this branch